cmd-recv.c

```diff
@@ -447,6 +447,20 @@ static void recv_trace_end(int sock, int efd)
 	close(sock);
 }
 
+static void execute_recv_cmd(char **argv) {
+	if (!argv)
+		return;
+
+	int pid = fork();
+	if (pid < 0)
+		pr_err("cannot start child process");
+
+	if (pid == 0) {
+		execvp(argv[0], argv);
+		pr_err("Failed to execute '%s'", argv[0]);
+	}
+}
+
```

Comment:
It's not work with likes "signal(SIGCHLD, wait_child)" because it's crashed on epoll_wait.
So, I added SIGCHLD to be handled on epoll_wait and check signal type.